### DISCUSSION TITLE
Is it possible to use an org file directly?

### DISCUSSION THREAD

**Discussion by lindoman** [200ok-ch/organice#905] [Q&A] [answered]
Totally basic question, maybe I misunderstand how this whole thing works. But can Organice use a file directly from local file storage without signing into some sort of back-end sync? I don't use any of those online sync options.

I am attempting this on Android... does that make a difference?

**Comment by schoettl** [comment, marked as answer]
No, that is not possible today. Browsers (in which organice run) generally don't support access to local filesystem. See also #663

**Reply by lindoman** [reply]
Thank you for the quick reply :)

**Reply by munen** [reply]
Schoettl is right, but you asked if running it on Android makes a difference. It does. Here’s documentation on how to set up a local WebDAV server: https://github.com/200ok-ch/organice/tree/master/contrib/organice-on-android-with-webdav-server

Glhf!

**Reply by schoettl** [reply]
#93 #373 #819 #730 – definitely an top-FAQ ^^

**Reply by lindoman** [reply]
Thanks, @munen I did try to set up WebDav already on Android, but it didn't work too good for me, I probably didn't do it right. I will try again with those instructions, thank you both.

**Comment by schoettl** [comment]
@munen Maybe we add "As a web app, it cannot sync with the local file system (yet)." under the first paragraph in [What does this project do?](https://github.com/200ok-ch/organice#what-does-this-project-do)?

**Reply by munen** [reply]
Since it's a common question, it sounds like a good idea. On the other hand, organice is becoming a popular project and popular projects always have weird questions coming in. If you state it does A, B and C, people will always want to have X, Y and Z. At Lambda Zen Temple, I have created a *huge* FAQ over time. Do people read it? Yes. But only those that wouldn't have come up with the weird question anyway. The others don't bother and just want X, Y and Z. People don't change(;

**Reply by lindoman** [reply]
As a total newcomer to this great piece of software, it was not clear to me exactly, and it is one of the first things I wanted to find out. I don't think it is a case of "I just want X Y Z" exactly, because org files are "usually" local files, if you see what I mean... Not that I get to vote on this ;) but I think a little up front explanation of what the implications are for being a web app, in relation to the use of local files, would be a useful addition (in simple English for those who like me, did not understand the implications)

**Reply by munen** [reply]
I am all for good documentation including a little up front explanation of what organice does. That’s why the docs begin with:

![29F57289-A9EA-48FB-B5D7-1484B21D8A9E](https://user-images.githubusercontent.com/505721/203418429-04823b86-d41c-47d5-803b-18875e018ea5.jpeg)

So, it syncs with (let’s say for brevity) A, B and C. It doesn’t state anything about X (local files), Y or Z (Syncthing). It’s hard to exclude all the things it does not or cannot do, because well… that is actually most things in the universe. Better to state what it does do, instead.

**Reply by lindoman** [reply]
Good point... :)

**Reply by lindoman** [reply]
But "it syncs with ..." didn't immediately say the same thing to me as "it only works if you use ..."

**Reply by munen** [reply]
Because it does not. Synching is built in a strategy pattern and anyone is free to build any technically possible strategy on top.

**Reply by schoettl** [reply]
I still think it's a important question that's worth to be addressed. Working with local files is so common and there is a demand for it. People with no background in the limits of web apps will continue to ask if or why it's not possible to "just simply open a local file".

I think adding a sentence stating why it's not possible could clear things up and wouldn't hurt. It would also add the information that it's a web app (which is no redundant information in this section). People maybe coming from outside, hopefully; some might have a totally different level of IT knowledge.
